src/schema/submission.yaml

```diff
@@ -165,9 +165,8 @@ classes:
     description: >-
       An condition that is linked to comparable samples.
     slots:
-      - title
```

Comment:
Shall we unify this through the model? Like either name the other title slots in different classes as "name", or the "name"s as title.